docs/020-deployment/howtos/040-configure-email.md

```diff
@@ -39,19 +42,21 @@ CLUEDIN_EMAIL_USER             (default: <blank>)
 
 ## Email Configuration - Kubernetes
 
-WHen using Kubernetes the SMTP setting can be configured in the `values.yaml`. This can be done by setting the following properties:
+When using Kubernetes the SMTP setting can be configured in the `values.yaml`. This can be done by setting the following properties:
```

Comment:
```suggestion
When using Kubernetes, the SMTP setting can be configured in the `values.yaml`. This can be done by setting the following properties:
```